Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
```

Comment:
What about to create `docker/php/symfony.ini` file and [COPY](https://docs.docker.com/reference/builder/#copy) it to the container instead of echoing with RUN command?